### PR TITLE
[DI] Allow service subscribers to leverage autowiring to know where their locator should be injected

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 3.3
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Symfony/Component/DependencyInjection/Tests/Dumper/PhpDumperTest.php [hunk 1]
```diff
@@ -13,6 +13,7 @@
 
 use DummyProxyDumper;
 use PHPUnit\Framework\TestCase;
+use Psr\Container\ContainerInterface as PsrContainerInterface;
```
Comment (dunglas) [inline src/Symfony/Component/DependencyInjection/Tests/Dumper/PhpDumperTest.php]: Why do you need to use an alias?

**Review by dunglas — APPROVED** [review]
👍

File: src/Symfony/Component/DependencyInjection/Tests/Dumper/PhpDumperTest.php (hunk 1 above)
Comment (nicolas-grekas) [inline src/Symfony/Component/DependencyInjection/Tests/Dumper/PhpDumperTest.php]: fixed

**Comment by weaverryan** [comment]
👍

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

**Pull request merged**